libs/libcds/tests/performance/libcds_hazard_pointer_overhead.cpp

```diff
@@ -342,7 +300,7 @@ int main(int argc, char* argv[])
 
     // clang-format off
     cmdline.add_options()
-        ("futures", value<std::uint64_t>()->default_value(500000),
+        ("futures", value<std::uint64_t>()->default_value(50000),
```

Comment:
@biddisco reduced the number in order to reduce testing time, maybe he can answer it. 

But to my understanding, launching more futures (i.e. 50k or 500k) than concurrently allowed attached threads (128 by default) is not a helpful testing, because the rest of threads (if there is already 128 threads running inside Hazard Pointer containers at one moment, and then the rest is 50k - 128) are not allowed to go into hazard pointer. So testing 128 futures might be good enough.

Reply:
It will be great for you to add a callback where appropriate, as I am not sure what does the callback looks like and how to add it.

Reply:
In other tests we use different number for Debug and Release configurations. One is for performance measurements, the other one for making sure things actually work without having to run for a long time.